CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/compose/theme/Color.kt

```diff
@@ -0,0 +1,8 @@
+@file:Suppress("MagicNumber", "TopLevelPropertyNaming")
+
+package co.nimblehq.coroutine.ui.screens.compose.theme
+
+import androidx.compose.ui.graphics.Color
+
+val BlueFreeSpeech = Color(0xFF3F51B5)
+val AlmostWhite = Color(0xFFDFE4EA)
```

Comment:
the variable name should not start with the capital word :neckbeard: You can create a class with static objects, then use `Color.blueFreeSpeech` instead 🛩️

Reply:
According to the official example of Jetpack Compose, they're all doing it like this. ⬇️  Do you think we should then follow their new convention in this case? 🤔 

<img width="509" alt="Screenshot 2021-07-26 at 17 10 59" src="https://user-images.githubusercontent.com/13492460/126972439-c510ba09-e3e7-4e40-a28f-6e3cb0c23163.png">

Reply:
@chornerman I see 🤔 it's okay to follow that naming convention. Besides, we could create a class to group the assets together, like `Color.Transparent`. How do you think?

Reply:
@manh-t Good idea. 😃  Will do 🙏🏼

Reply:
Resolved 🙏🏼